src/engine/source/metrics/input_files/metrics-config.json

```diff
@@ -1,76 +1,124 @@
 [
     {
-        "name": "MetricsTestCounter",
+        "name": "Server.ReceivedBytesPerSeconds",
+        "dataHubEnable" : true,
+        "enable": true,
+        "signalType": "meter",
+        "instrumentType" : "counter",
+	    "aggregationTemporality" : "delta",
+        "subType": "uint64",
+	    "exportIntervalMillis": 1000,
+        "exportTimeoutMillis": 500,
+        "unit": "bytes/second",
```

Comment:
Verify the unit based on the official [documentation](https://opentelemetry-cpp.readthedocs.io/en/latest/).
We have to use the following [standard](https://unitsofmeasure.org/ucum).